IMC_DataScience/IMC_OOPTutorial/solution/Person.java

```diff
@@ -0,0 +1,47 @@
+public class Person
+{
+	//Coding this will be an exercise for the students participating in the lesson; incomplete code fragments will be used to initially demonstrate the concepts.
+	//As this is a simple program, there isn't any comments within the body - the method and variable names are intended to be self-explanatory.
+	
+	private String firstName;
+	private String lastName;
+	private int age;
+
+	public Person(String theirFirstName, String theirLastName, int theirAge)
+	{
+		firstName = theirFirstName;
+		lastName = theirLastName;
+		age = theirAge;
```

Comment:
Add a checker for age so that we can't input a negative age or a blank first/last name